gulpfile.js

```diff
@@ -33,10 +32,6 @@ gulp.task('js:app', function (){
         ;
 });
 
-gulp.task('webserver', function() {
```

Comment:
Hm yes no problem. Seems to be changed in the merging because I had some conflicts, this branch is very old.